spec/features/invoices/show_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+
+describe 'invoices show page', js: true do
+  include MoneyHelper
+
+  let(:user) { create :user }
+  let(:org)  { create :organization, with_user: user }
+
+  before do
+    sign_in user
+  end
+
+  subject{ page }
+
+  context 'Download as PDF' do
+    let!(:invoice) { create :invoice, organization: org }
+
+    before do
+      visit invoice_path(invoice)
+      click_on 'Download as PDF'
+    end
+
+    it 'PDF content' do
```

Comment:
@yunixon I suggest to test usual PDF downloading at first, check downloaded file name, file size (greater than nil). And in second case you can just do `visit invoice_path(@invoice, {format: :pdf, debug: true})` and check PDF content, as you do here.